the_destination_of_shadow/Draw.cpp

```diff
@@ -117,7 +116,7 @@ VOID DrawTex::LoadTexture(const CHAR * file_name, INT TEX)
 }
 
 
-DOUBLE DrawTex::to_Rad(DOUBLE degree)
+DOUBLE Texture::to_Rad(DOUBLE degree)
```

Comment:
なんかこれだけ他と命名が合わないですね。

```suggestion
DOUBLE Texture::ToRadian(DOUBLE degree)
```
かな。

Reply:
修正しました